scanpy-scripts-tests.bats

```diff
@@ -1,6 +1,4 @@
-#!/usr/bin/env bats
```

Comment:
Removing these lines is not a good plan!

Reply:
Also please recover the permission of this script to `755` so that it's executable.